chains/solana/contracts/programs/ccip-router/src/messages.rs

```diff
@@ -356,14 +356,22 @@ impl Solana2AnyMessage {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
Is this still flagged as dead code by the linter, or could we remove the attribute?

Reply:
Ah I missed this, sorry!